### PR TITLE
Release note for Poison Pill 4.9

### PR DESCRIPTION
**Pull request by abhatt-rh**
https://issues.redhat.com/browse/TELCODOCS-343: RN for Poison Pill Operator 

Preview: https://deploy-preview-41180--osdocs.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-9-release-notes#ocp-4-9-poison-pill-operator
 
 Note: The link in the release note will be active after the [main docs PR for Poison Pill 4.9](https://github.com/openshift/openshift-docs/pull/40380) is merged.

### CODE REVIEW

**Comment by abhatt-rh** [comment]
Hi @shellymiron @prabinovRedhat @mshitrit,
Request you to kindly review the Release note for Poison Pill 4.9

**Comment by abhatt-rh** [comment]
Hi @beekhof, PTAL at the release note for Poison Pill 4.9

**Review by beekhof — APPROVED** [review]

File: release_notes/ocp-4-9-release-notes.adoc [hunk 1]
```diff
@@ -734,6 +734,14 @@ When configuring the `docker/config.json` file to allow pods to pull images from
 
 Node-related metrics and alerts have been enhanced to give you an earlier indication of when the stability of a node is compromised.
 
+[id="ocp-4-9-poison-pill-operator"]
+==== Enhanced remediation with the Poison Pill Operator 
+
+The Poison Pill Operator now provides following new capabilities:
```
Comment (ahardin-rh) [inline release_notes/ocp-4-9-release-notes.adoc:740]: s/provides following new/provides these new

File: release_notes/ocp-4-9-release-notes.adoc [hunk 2]
```diff
@@ -734,6 +734,14 @@ When configuring the `docker/config.json` file to allow pods to pull images from
...
+
+* Independent of cluster installation type, such as Installer Provisioned Infrastructure (IPI)or User Provisioned Infrastructure (UPI).
```
Comment (ahardin-rh) [inline release_notes/ocp-4-9-release-notes.adoc:742]: What does being independent of installation type mean? 
Need space after (IPI)

Comment (abhatt-rh) [reply release_notes/ocp-4-9-release-notes.adoc]: s/Independent of/regardless of
and rephrased it to align it with the description in the main docs content.

**Comment by ahardin-rh** [comment]
LGTM! Thank you!

**Pull request merged**